### PR TITLE
Rename SubscriptionOpts to BlockRange

### PR DESCRIPTION
**Pull request by LogvinovLeon**
This PR:
* Renames the type (and schema) to better represent it's content

### CODE REVIEW

File: packages/0x.js/src/contract_wrappers/exchange_wrapper.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ import {
     OrderTransactionOpts,
     OrderValues,
     SignedOrder,
-    SubscriptionOpts,
+    BlockRange,
```
Comment (BMillman19) [inline packages/0x.js/src/contract_wrappers/exchange_wrapper.ts]: alphabetize

File: packages/0x.js/src/contract_wrappers/exchange_wrapper.ts [hunk 2]
```diff
@@ -610,20 +610,20 @@ export class ExchangeWrapper extends ContractWrapper {
     /**
      * Gets historical logs without creating a subscription
      * @param   eventName           The exchange contract event you would like to subscribe to.
-     * @param   subscriptionOpts    Subscriptions options that let you configure the subscription.
+     * @param   blockRange    Subscriptions options that let you configure the subscription.
```
Comment (BMillman19) [inline packages/0x.js/src/contract_wrappers/exchange_wrapper.ts]: spacing

File: packages/0x.js/src/index.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ export {
     ExchangeEvents,
     TokenEvents,
     IndexedFilterValues,
-    SubscriptionOpts,
+    BlockRange,
```
Comment (BMillman19) [inline packages/0x.js/src/index.ts]: can we alphabetize this entire list? I thought the linter would enforce this

File: packages/0x.js/test/exchange_wrapper_test.ts [hunk 4]
```diff
@@ -13,7 +13,7 @@ import {
     OrderCancellationRequest,
     OrderFillRequest,
     SignedOrder,
-    SubscriptionOpts,
+    BlockRange,
```
Comment (BMillman19) [inline packages/0x.js/test/exchange_wrapper_test.ts]: same here

File: packages/0x.js/test/token_wrapper_test.ts [hunk 5]
```diff
@@ -8,7 +8,7 @@ import * as Web3 from 'web3';
 import {
     ApprovalContractEventArgs,
     DecodedLogEvent,
-    SubscriptionOpts,
+    BlockRange,
```
Comment (BMillman19) [inline packages/0x.js/test/token_wrapper_test.ts]: here too

File: packages/0x.js/src/contract_wrappers/exchange_wrapper.ts [hunk 6]
```diff
@@ -610,20 +610,20 @@ export class ExchangeWrapper extends ContractWrapper {
     /**
      * Gets historical logs without creating a subscription
      * @param   eventName           The exchange contract event you would like to subscribe to.
-     * @param   subscriptionOpts    Subscriptions options that let you configure the subscription.
+     * @param   blockRange          Subscriptions options that let you configure the subscription.
```
Comment (fabioberger) [inline packages/0x.js/src/contract_wrappers/exchange_wrapper.ts]: Change the description here too.

File: packages/0x.js/src/contract_wrappers/token_wrapper.ts [hunk 7]
```diff
@@ -285,20 +285,20 @@ export class TokenWrapper extends ContractWrapper {
      * Gets historical logs without creating a subscription
      * @param   tokenAddress        An address of the token that emmited the logs.
      * @param   eventName           The token contract event you would like to subscribe to.
-     * @param   subscriptionOpts    Subscriptions options that let you configure the subscription.
+     * @param   blockRange          Subscriptions options that let you configure the subscription.
```
Comment (fabioberger) [inline packages/0x.js/src/contract_wrappers/token_wrapper.ts]: Change description.

**Pull request merged**